addons/web/static/src/js/core/smooth_scroll_on_drag.js

```diff
@@ -356,6 +340,20 @@ const SmoothScrollOnDrag = Class.extend(mixins.ParentedMixin, {
      * @param {Function} onDragStartCallBack The jQuery drag callback.
      */
     _onSmoothDragStart(ev, ui, onDragStartCallBack) {
+        const elementOffset = $(ev.target).offset();
+        this.options.jQueryDraggableOptions.cursorAt = {
```

Comment:
Hello!
Previously the cursorAt was set that way only if none where passed in through the options. Why having changed that behavior?

Reply:
For a drag and drop in an iframe it didn't work.

Reply:
this branch is no longer used, it is the branch: `14.0-jabberwock-sge-age-dmo-chm-nby`